spec/models/user_spec.rb

```diff
@@ -39,6 +39,7 @@
 
     it { is_expected.to validate_presence_of :email }
     it { is_expected.to validate_uniqueness_of(:email).case_insensitive }
+    it { is_expected.to_not allow_value('not_a_valid_email').for(:email) }
```

Comment:
This should be `not_to` instead of `to_not`. (Code Climate *should* be flagging that, as our Rubocop configuration enforces the `not_to` format. I have a support ticket open w/ Code Climate to investigate the false negative.)

Reply:
Changed.